chandra_aca/maude_decom.py

```diff
@@ -355,9 +355,18 @@ def unpack_aca_telemetry(packet):
         img_pixels = np.sum(np.packbits(_pixel_bits, axis=1) * [[2 ** 8, 1]], axis=1)
         img_header['pixels'] = img_pixels
         slots.append(img_header)
-    integ, glbstat = _unpack('>HB', packet[:3])
+
+    # The first two bytes that were integration time will have first two bits for
```

Comment:
Write this comment so it will also work in 2027.  You can mention pre- / post-dynamic background patch.

Reply:
I don't quite understand the comment on the comment about 2027, but seems fine to remove the "were integration time" part.

Reply:
I went with 
```
    # Before the dynamic background patch, the first two bytes contained INTEG in those
    # 16 bits (named integbits).  After the dynamic background patch, the first 6 bits of
    # integbits will be repurposed: two bits for PIXTLM, next bit for BGDTYP, 3 spares,
    # and 10 bits for INTEG.  This telem/decom change is back-compatible and can be promoted
    # before the dynamic background patch is in use onboard.
```